core/tests/webdriverio_utils/waitFor.js

```diff
@@ -36,6 +36,19 @@ var alertToBePresent = async() => {
     });
 };
 
+// In some cases we explicitly needs to wait for url to change
+// as test takes some time to change the url and webdriverio
+// doesn't wait that much.
```

Comment:
```suggestion
// Wait for current url to change to a specific url
```
Please give a yes if this sounds good to you.

Reply:
Done.